jobs/config.json

```diff
@@ -9309,27 +9307,6 @@
       "sig-scalability"
     ]
   },
-  "ci-kubernetes-kubemark-5-prow-canary": {
```

Comment:
Instead of removing the job, could you make its config match the regular kubemark-5 job?